index.html

```diff
@@ -1,83 +1,96 @@
-    <link rel="stylesheet" href="style.css">
-    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon" />
     <title>Google</title>
-</head>
-<body>
-
+  </head>
+  <body>
     <div class="userinfo">
-        <ul>
-            <li> <a href="#">Gmail</a></li>
-            <li> <a href="#">Images</a></li>
-            <li> 
-                <div class="tooltip">
-                    <img src="./images/appGoogle.png" alt="google app" class="googleapp"><span class="toolTipText"> Google apps</span>
-                </div>
-                </li>
-            <li> 
-                <div class="tooltipletter">
-                    <span class="letterborder">K</span> <span class="tooltipemail"><span class="ga"> Google Account</span> <br/> Kanu Mike Chibundu <br/> Michotall95@gmail.com</span>
-                </div>
-            </li>
-        </ul>
+      <ul>
+        <li><a href="#">Gmail</a></li>
+        <li><a href="#">Images</a></li>
+        <li>
+          <div class="tooltip">
+            <img
+              src="./images/appGoogle.png"
+              alt="google app"
+              class="googleapp"
+            /><span class="toolTipText"> Google apps</span>
+          </div>
+        </li>
+        <li>
+          <div class="tooltipletter">
+            <span class="letterborder">K</span>
+            <span class="tooltipemail"
+              ><span class="ga"> Google Account</span> <br />
+              Kanu Mike Chibundu <br />
+              Michotall95@gmail.com</span
+            >
+          </div>
+        </li>
+      </ul>
     </div>
 
     <div class="headGoogle">
-        <img src="./images/googleimg.png" alt="google image" class="googleimg">
-        <!-- <h1><span class="gcolor">G</span><span class="firsto">o</span><span class="secondo">o</span><span class="gcolor">g</span><span class="lcolor">l</span><span class="firsto">e</span></h1> -->
+      <img src="./images/googleimg.png" alt="google image" class="googleimg" />
+      <!-- <h1><span class="gcolor">G</span><span class="firsto">o</span><span class="secondo">o</span><span class="gcolor">g</span><span class="lcolor">l</span><span class="firsto">e</span></h1> -->
     </div>
 
     <div class="googlesearch">
-        <div class="imgsearch"> <img src="./images/search.png" alt="search icon"></div>
-        <form action="" class="formalign">
-            <input type="search" name="search" class="search">
-        </form>
-        <div class="imgmic"> <img src="./images/microphone.png" alt="microphone"></div>
+      <div class="imgsearch">
+        <img src="./images/search.png" alt="search icon" />
+      </div>
```

Comment:
Please use a section html tag for line 38 and close with the corresponding section tag

Reply:
- [ ] It is always better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: *Why don't you use an article/section/aside/etc tag instead of a div, that would improve the HTML structure and make the page more accessible*

This request lacks a "why": *Why don't you use an article tag here instead of a div?*


- [ ] It is also a good practice to be as clear as possible in your review. Try as much as possible to not confuse the code author. 
For example, from the codes you highlighted, line 38 does not require a `section` tag since it's just an `input` tag inside a `form` tag. Please be specific with the line of code you are requesting for changes.